templates/zerver/register.html

```diff
@@ -93,6 +93,7 @@ <h1>You&rsquo;re almost there.</h1>
                 <div class="input-box no-validation">
                     <input type='hidden' name='key' value='{{ key }}' />
                     <input type='hidden' name='timezone' id='timezone'/>
+                    <input type='hidden' name='time_format' id='time_format'/>
```

Comment:
Basically this part could be `twenty_four_hour_time` and we could use this field as bool rather than a string.

Reply:
ok